graphql_compiler/compiler/emit_sql.py

```diff
@@ -879,10 +904,15 @@ def _join_to_parent_location(
                 )
             )
 
+        on_clause = sqlalchemy.and_(*(
+            parent_alias.c[from_column] == self._current_alias.c[to_column]
+            for from_column, to_column in matching_column_pairs
+        ))
```

Comment:
You've probably tested this already, so please excuse my ignorance — what does `sqlalchemy.and_` do when the list has only one element? Presumably it outputs it as-is?

But what about having no elements? And is that ever possible?

Reply:
It's the identity function for 1 argument. Old tests would fail if it did something else.

I'll add an assertion to check for 0 elements.